server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -56,6 +56,7 @@
 public class SnapshotsInProgress extends AbstractNamedDiffable<Custom> implements Custom {
```

Comment:
Maybe adjust `SnapshotsInProgressSerializationTests`, so that a randomly generated instance also contains randomly a list of data stream names.

Reply:
Done